<launch>

  <node pkg="labeled_object_detector" type="detect_nearest_object_action" output="screen" respawn="true" name="object_detector"
>
    <remap from="build_cloud" to="/pcd_assembler/build_cloud" />
    <param name="fixed_frame" type="string" value="map" />
    <param name="local_frame" type="string" value="odom_combined" />

  </node>


</launch>